content/en/docs/concepts/scheduling-eviction/kube-scheduler.md

```diff
@@ -32,11 +32,7 @@ is the default scheduler for Kubernetes and runs as part of the
 kube-scheduler is designed so that, if you want and need to, you can
 write your own scheduling component and use that instead.
 
-For every newly created pod or other unscheduled pods, kube-scheduler
-selects an optimal node for them to run on. However, every container in
-pods has different requirements for resources and every pod also has
-different requirements. Therefore, existing nodes need to be filtered
-according to the specific scheduling requirements.
+Kube-scheduler selects an optimal node to run newly created or not yet scheduled (unscheduled) pods. Since containers in pods, and pods themselves have different requirements, existing nodes need to be filtered according to the specific scheduling requirements.
```

Comment:
How about:
```suggestion
To run a Pod, Kubernetes needs to know which node that Pod will run on.
Although the API lets you specify a node for a Pod when you create the Pod,
this is unusual and is only done in special cases. Instead, the kube-scheduler
tries to work out which node is the best choice to for placing that new Pod.
Since containers in pods - and pods themselves - can have different requirements,
the schedule filters out any nodes that don't meet a Pod's specific scheduling
needs.
```